vertexai/src/GenerativeModel.cs

```diff
@@ -183,7 +187,7 @@ private async Task<CountTokensResponse> CountTokensAsyncInternal(
   }
 
   private string GetURL() {
-    return "https://firebaseml.googleapis.com/v2beta" +
+    return "https://firebasevertexai.googleapis.com/v1beta" +
```

Comment:
Shouldn't this be a constant at the top of the file somehow?

Reply:
Maybe, but since it is only used here, and also it seems like this logic is going to have to be rewritten in the nearish future anyway with different url layouts based on different scenarios, I'm just going to leave it as is for now.